sdk/core/azure-core-test/inc/azure/core/test/test_context_manager.hpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * This class handles managing context about a test.
+ */
+
+#pragma once
+
+#include <gtest/gtest.h>
+#include <string>
+
+#include "azure/core/test/network_models.hpp"
+
+namespace Azure { namespace Core { namespace Test {
+  class TestContextManager {
+  public:
+    std::string RecordingPath;
+    Azure::Core::Test::TestMode TestMode;
+    bool LiveOnly = false;
+
+    TestContextManager() { SetLiveOnly(); }
+
```

Comment:
comments

Reply:
all this package is not expected to be published or exposed to customers. 
We won't generate docs to be published or so.

I am happy to add comments anyway for consistency, but the comments are just for ourselves